### PR TITLE
feat: Add i18n support and support Chinese

### PR DESCRIPTION
**Pull request by qiwihui**
Main changes:
- add vue-i18n to support international;
- add Chinese support;
- determine language by browser navigator settings, default is English.

### CODE REVIEW

**Comment by qiwihui** [comment]
refer #45

**Comment by cars10** [comment]
Hey @qiwihui , thank you for your contribution to elasticvue! I am still on vacation right now but i will look into this in the following week.

File: src/views/Utilities.vue [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-          <h1 class="text-h5">Utilities</h1>
+          <h1 class="text-h5">{{ $t("utilities.title") }}</h1>
         </v-card-title>
         <v-divider/>
 
         <v-card-text>
           <v-row>
             <v-col cols="12" md="6" sm="12">
-              <v-subheader class="text-h6">Create</v-subheader>
+              <v-subheader class="text-h6">{{ $t("utilities.title") }}</v-subheader>
```
Comment (cars10) [inline src/views/Utilities.vue:13]: I guess this should be `utilities.create.title`

**Comment by cars10** [comment]
Thank you for putting in the work to add i18n to elasticvue :)
I am mostly fine with the code and structure. 
My only concern is that maintaining i18n is an ongoing effort, especially when i don't speak the language.  Thinking of future features, text changes, etc: how good are the results for google translate in chinese? Is it possible for someone that does not speak it (me) to generate somewhat usable translations for chinese?

**Comment by qiwihui** [comment]
Yes, that's really a tough problem need to be considered. Currently, the string that does not match locales string will fallback to English. I don't have much experience in managing i18n files. Is there any good practice?

**Comment by cars10** [comment]
> Yes, that's really a tough problem need to be considered. Currently, the string that does not match locales string will fallback to English. I don't have much experience in managing i18n files. Is there any good practice?

Best practice would obviously be to always add the complete i18n when adding/changing features/texts. But i guess for chinese i will have to rely on PRs that periodically update the i18n, so the chinese i18n will most likely always be at least 1 version "behind" and fallback to english for new features. Not ideal, but i don't really see an alternative.

**Comment by cars10** [comment]
So far i will leave this PR open until the next version of elasticvue is ready, because there are still some text changes/additions to come. It would be awesome if you could translate those too so we have a complete chinese translation at least once :) I will ping you again as soon as i am done with the version. Thank you for the awesome work so far!

Apart from that i would like to propose some best practices for i18n structuring that we use quite successfully at my current job:
Key should be named like this: `path.to.component.component_name.optional_structuring.key_name`. 
For example the text `"Chose instance"` in `components/ElasticsearchInstance/ElasticsearchInstanceSelection.vue` would be named: `elasticsearch_instance.elasticsearch_instance_selection.chose_instance`. The keys *do* get quite long in some cases, but you always know exactly where your key is used. Something like `es.instance` is nice and short but does not really tell you anything about the usage of the key. This also implies that there *will be* duplication and keys with the same values, this is expected and wanted. Because if you know that every key i used exactly once then you can safely rename/remove a key without having to worry that you just broke some part.

The only exception are keys that are used many times, like the placeholder `"Filter..."` in most filter inputs. We do use a subkey called `defaults.` for these kind of things, but its hard to keep this clean and not stuff everything in there that is used two times or more.

This is not something that i expect you to change in your current implementation, this can easily be done by me later with find&replace. I just wanted to share some of our experience :) Great work and thanks again for the PR!

**Comment by cars10** [comment]
@qiwihui i guess this version is mostly done. It would be great if you could go other the changes on the develop branch and add the remaining chinese translations, thanks a lot!

**Comment by qiwihui** [comment]
@cars10 I will finish it within 24 hours in order not to block your release.

**Comment by qiwihui** [comment]
@cars10 I will finish it within 24 hours in order not to block your release.

**Comment by qiwihui** [comment]
@cars10 All has been finished. Please help to review this PR.

**Comment by cars10** [comment]
LGTM! I will merge this later today. Might make a few smaller adjustments after the merge, but this is an awesome PR. Thanks :)

**Comment by cars10** [comment]
@qiwihui i went trough the project and found some more i18n that we missed in the first run. `cn.json` is missing about 50 translations, they are all set to `null` right now so we can easily find them. It would be awesome if you can help me again to translate these :)

**Comment by qiwihui** [comment]
@cars10 Ok, it is done. Please help to review it.

**Pull request merged**